pkg/apis/kops/cluster.go

```diff
@@ -486,3 +487,17 @@ func (z *ClusterZoneSpec) assignCIDR(c *Cluster) (string, error) {
 func (c *Cluster) SharedVPC() bool {
 	return c.Spec.NetworkID != ""
 }
+
+// --------------------------------------------------------------------------------------------
+// Network Topology functions for template parsing
+//
+// Each of these functions can be used in the model templates
+// The go template package currently only supports boolean
```

Comment:
I think we should continue to use the templates and follow suit here - if we want to move away from them later we can.. But I think that is a larger effort.

They work for now, so lets just keep it consistent